app/Http/Controllers/Assets/BulkAssetsController.php

```diff
@@ -90,6 +91,8 @@ public function update(Request $request)
             || ($request->filled('model_id'))
         ) {
             foreach ($assets as $assetId) {
+                \Log::error($assetId);
```

Comment:
This looks like leftover debugging - fine with setting it to `Log::debug` if you want to keep it.